app/controllers/concerns/idv/verify_info_concern.rb

```diff
@@ -196,27 +197,13 @@ def log_idv_verification_submitted_event(success: false, failure_reason: nil)
     end
 
     def check_ssn
-      result = Idv::SsnForm.new(current_user).submit(ssn: pii[:ssn])
-
-      if result.success?
-        save_legacy_state
-        delete_pii
-      end
-
-      result
+      Idv::SsnForm.new(current_user).submit(ssn: pii[:ssn])
     end
 
-    def save_legacy_state
-      skip_legacy_steps
+    def move_applicant_to_idv_session
```

Comment:
Is there a method name that would explain why this happens? Maybe `idv_session.save_pii` rather than setting idv_session properties from the outside.

Reply:
I'm not sure about adding a `#save_pii`. Wouldn't that just be an alias for `#applicant=`?

Reply:
`#save_pii` might not be the best name. The general thought is to call methods on `Idv::Session` that describe the purpose, like marking steps complete or invalidating them, rather than setting specific properties.